nova_plugin/server.py

```diff
@@ -253,7 +253,12 @@ def rename(name):
     ctx.logger.debug(
         "server.create() server before transformations: {0}".format(server))
 
-    _handle_image_or_flavor(server, nova_client, 'image')
+    if 'block_device_mapping' in server:
```

Comment:
Ah... unless you're saying we should support the user supplying the volume mapping directly in the `server` input?
Do we need to do that, given we're providing a way to do it with a relationship?